src/main/scala/mesosphere/marathon/upgrade/TaskReplaceActor.scala

```diff
@@ -26,7 +26,9 @@ class TaskReplaceActor(
 
   val tasksToKill = taskTracker.marathonAppTasksSync(app.id)
   val appId = app.id
-  val version = app.version.toString
+  val version = app.version
+  // FIXME: Remove!
```

Comment:
I shouldn't have conflated this PR with changing Timestamp strings to Timestamps. I created an issue https://github.com/mesosphere/marathon/issues/3207 and I will remove the FIXME. (Otherwise I'll have to create a lot of FIXMEs ;) )

The issue contains specific instructions on how to find these occurrences. Prioritize as you like.